docs/04.guides/08.lucee-5/01.extensions/page.md

```diff
@@ -30,9 +30,30 @@ You can install extensions by declaring the "id" of the extension as a JVM argum
 ## Convention Based ##
 The "convention based" approach means you simply have folders for different parts of your application, for example:
 
-/applications - for cfm & cfc files that need to be installed in the webroot
-/functions - for build-in functions to install
-/jars - for libraries used by your extension
+- /applications - for cfm & cfc files that need to be installed in the webroot
+- /functions - for built-in functions to install
+- /jars - for libraries used by your extension
+- /plugins/{plugin name}/* - for adding an admin plugin
+
+The complete list of directories and file extensions which you can add to a .lex archive is (distilled from [the source code](https://github.com/lucee/Lucee/blob/master/core/src/main/java/lucee/runtime/config/XMLConfigAdmin.java#L4527)):
+`/jars/*.jar`
```

Comment:
Might be good to make this a bullet list. These single newlines in .md will actually result in one big paragraph that's not so readable.